eslint.config.mjs

```diff
@@ -48,47 +57,43 @@ export default [
 
     settings: {
       node: {
-        version: '>=16.0.0'
+        // Used by `eslint-plugin-n` to determine the minimum version of Node.js to support.
+        // Normally setting this in the `package.json` engines field is enough, but when we have more than one active
+        // major release line at the same time, we need to specify the lowest version here to ensure backporting will
+        // not fail.
+        version: '>=18.0.0'
```

Comment:
didn't we agree we'd keep v4 "backportable" in case we need to, and thus keep v16.0.0 for now ?

Reply:
> didn't we agree we'd keep v4 "backportable" in case we need to, and thus keep v16.0.0 for now ?

Given a few recent commits completely dropping support for it, I think the hope is that we'd be able to only backport critical fixes if absolutely necessary, so this PR wouldn't apply.